cypress/integration/api.js

```diff
@@ -0,0 +1,9 @@
+describe('API component', () => {
+   beforeEach(() => {
+       cy.visit("http://localhost:4200/");
+   });
+
+   it('Loads correctly', () => {
+       cy.get("h1:contains('Introduction')").should('exist');
+   })
+});
```

Comment:
Ending newline? Why does eslint not format these? 😶

Reply:
No clue! :O